ipld-future/README.md

```diff
@@ -25,20 +25,20 @@ Editor: Nicola Greco, MIT
 ## Introduction
 Naming things with hashes solves three fundamental problems for the decentralized web:
 
-1. **Data integrity**: URLs give no guarantees that the data we receive hasn't been compromised, the IPLD naming system ensures that no one can lie about the data they are serving
-2. **Distributed naming**: only the owner of a domain name can serve you the data behind a URL; in IPLD any computer - trusted and untrusted - that has the data can help and participate in distributing it
-3. **Immutable Content**: The content behind URLs can change or disappear, making our links broken or not pointing anymore to what we were expecting. IPLD links cannot mutate.
+1. **Data integrity**: URLs give no guarantees that the data we receive hasn't been compromised. The IPLD naming system ensures that no one can lie about the data they are serving.
```

Comment:
thank you!
Previous work especially the RFC on "NI" was referred to as "naming things with hashes".

We can choose either wording and stick with one, I like addressing since naming systems are often confused with DNS replacements